registration/views.py

```diff
@@ -318,7 +334,7 @@ def process_registration_configuration(self):
 
     def submit_registration(self, data):
         try:
-            FIFTYTHREE_CLIENT.register(**data)
+            FIFTYTHREE_CLIENT.register(get_external_source_data(self.request.session), **data)
```

Comment:
@waseem-omar as indicated above, I kinda think we should incorporate the external source data into the data passed to ``register`` here. Something like:

    data_copy = data.copy()
    data_copy.update(get_external_source_data(self.request.session))